Readability.js

```diff
@@ -119,6 +119,36 @@ Readability.prototype = {
   },
 
   /**
+   * Iterate over a NodeList, which doesn't natively fully implement the Array
+   * interface.
```

Comment:
The downside of forEach used to be that the extra function calls produced overhead, so the performance is worse. I don't know if that's still (meaningfully) the case.